crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -505,12 +505,23 @@ impl Default for GpuClusterableObjectsUniform {
 //   clusterable objects are chosen if the clusterable object count limit is
 //   exceeded.
 pub(crate) fn clusterable_object_order(
-    (entity_1, shadows_enabled_1, is_spot_light_1): (&Entity, &bool, &bool),
-    (entity_2, shadows_enabled_2, is_spot_light_2): (&Entity, &bool, &bool),
+    (entity_1, shadows_enabled_1, is_volumetric_light_1, is_spot_light_1): (
```

Comment:
Nit: This collection of bools is starting to get confusing; maybe refactor this into a private helper struct?